docs/runtimes/configuring-job-resources.md

```diff
@@ -237,22 +231,21 @@ And another function that can only be scheduled on preemptible nodes:
 import mlrun
 import os
 
-train_fn = mlrun.code_to_function('training', 
+train_fn = mlrun.set_function('training', 
                             kind='job', 
                             handler='my_training_function') 
 train_fn.with_preemption_mode(mode="prevent") 
 train_fn.run(inputs={"dataset": my_data})
 ```
 
-See {py:meth}`~KubeResource.with_preemption_mode.
-
-Alternatively, you can specify the preemption using `with_priority_class` and `with_node_selection` parameters. This example specifies that 
+Alternatively, you can specify the preemption using {py:meth}`~mlrun.runtimes.KubeResource.with_priority_class` and 
+{py:meth}`~mlrun.runtimes.KubeResource.with_node_selection` parameters. This example specifies that 
 the pod/function runs only on non-preemptible nodes:
 
 ```
 import mlrun
 import os
-train_fn = mlrun.code_to_function('training', 
+train_fn = mlrun.set_function('training', 
                             kind='job', 
                             handler='my_training_function') 
 train_fn.with_preemption_mode(mode="prevent") 
```

Comment:
If we're working with priority-class and node-selection, why do we need `with_preemption_mode`? It's redundant and possibly conflicting. Looks like this line needs to be removed.